hutch_python/load_conf.py

```diff
@@ -206,9 +211,33 @@ def load_conf(conf, hutch_dir=None):
                          'load objects from questionnaire or experiment '
                          'file.'))
 
+    try:
+        # Configure whether we use the LCLS-I or LCLS-II daq
+        daq_type = conf['daq_type']
+        if daq_type in ('lcls1', 'lcls1-sim', 'lcls2', 'nodaq'):
```

Comment:
Nitpick - should move this out of the try/except as it won't be raising `KeyError` (or in `else:`)

Reply:
I think this is a difference in intent rather than a straight error.

The intent of the try/except is just to catch a missing key to provide the default. I don't want to fall back to the default if the user passes an invalid key, I want to make red text in the startup and not load the daq in the session. Config errors should be loud I think. Maybe this one should be even louder?

Reply:
My terse comment wasn't as clear as I intended - I meant "keep the `try` section small and only relevant to what could be raised; the first line raising `KeyError` is the only relevant one"

Reply:
Ah, I understand now. I'll move these into the often-forgotten try/except/else